swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/test-services-and-clients.md

```diff
@@ -56,12 +56,11 @@ public function testGet() returns error? {
 }
 ```
 
-## Test clients
+## Test with clients
 
-In cases where a fully fledged client is already defined for a particular service, you can make use
-of `object mocking` to mock the calls to the service and return curated responses to the client.
-It is useful when testing the full extent of the client by mocking responses that are difficult to reproduce in actual scenarios.
- It would cover a variety of cases that the client can handle without having the service to be up and running.
+In cases where a fully-fledged client is already defined for a particular service, and the client is used across your code. Now, you want to test your code. Here, you can make use
```

Comment:
`In cases where a fully-fledged client is already defined for a particular service, and the client is used across your code` This doesn't look like a proper sentence with a meaning.